recipes/griddataformats/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "GridDataFormats" %}
+{% set version = "0.3.3" %}
+{% set sha256 = "938f0efcb3bc2f58ec85048b933942da8a52c134170acc97cb095f09d3698fbd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - numpy
+    - scipy
+
+  run:
+    - six
+    - numpy
+    - scipy
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  imports:
+    - gridData
+
+about:
+  home: http://github.com/MDAnalysis/GridDataFormats
+  license: LGPLv3
+  license_file: COPYING.LESSER
+  summary: 'Reading and writing of data on regular grids in Python'
+
+  # The remaining entries in this section are optional, but recommended
+  description: |
+    The gridDataFormats package provides classes to unify reading and
+    writing n-dimensional datasets. One can read grid data from files,
+    make them available as a `Grid`_ object, and write out the data again.
+  doc_url: http://www.mdanalysis.org/GridDataFormats/
+  dev_url: https://github.com/MDAnalysis/GridDataFormats
+
+extra:
+  recipe-maintainers:
+    - kain88-de
+    - orbeckst
```

Comment:
@orbeckst and @richardjgowers -- are you ok with being added as co-maintainers of this recipe?